pages/edge/configuration/app-configuration.mdx

```diff
@@ -45,6 +45,13 @@ The `package` field is used to provide the primary workload to execute. It is **
 
 - Required: `true`
 
+### `debug`
+
+The `debug` field is used to enable debug mode for the application. This in turn will show detailed error messages if a request fails, instead of a generic the instance failed. It is **optional** and can be any boolean.
```

Comment:
```suggestion
The `debug` field is used to enable debug mode for the application. This in turn will show detailed error messages if a request fails, instead of a generic error. It is **optional** and can be any boolean.
```

Reply:
Also I would add:

> this should be turned of in production, but can be very helpful when developing.